students.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
```

Comment:
- [ ] please kindly ensure that When setting the classroom for a student it also adds it to the classrooms' students
`attr_reader :classroom`
- [ ] kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom)
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```